src/integrations/shared-invite-reply/parts/response-actions.tsx

```diff
@@ -33,7 +43,17 @@ const ResponseActions: FC<ResponseActionsProps> = ({
 	const [calendarName, setCalendarName] = useState(sharedCalendarName);
 	const [selectedColor, setSelectedColor] = useState(0);
 	const accounts = useUserAccounts();
-
+	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+	// @ts-ignore
+	const calFolders = useFoldersAccordionByView(FOLDER_VIEW.appointment, null);
```

Comment:
it looks like this hook is not working correctly, it will need to be fixed before merging